node/src/types/exit_code.rs

```diff
@@ -11,18 +11,14 @@ const SIGNAL_OFFSET: u8 = 128;
 /// Note that a panic will result in the Rust process producing an exit code of 101.
 #[derive(Clone, Copy, PartialEq, Eq, Debug, DataSize)]
 #[repr(u8)]
+#[non_exhaustive]
 pub enum ExitCode {
     /// The process should exit with success.  The launcher should proceed to run the next
     /// installed version of `casper-node`.
     Success = 0,
     /// The process should exit with `101`, equivalent to panicking.  The launcher should not
     /// restart the node.
     Abort = 101,
-    /// The process should exit with `102`.  It used to be an indication to the launcher
-    /// that it should proceed to run the previous installed version of `casper-node`.
-    /// It is no longer used, but we keep it here to avoid it being reassigned to other features.
```

Comment:
I would advise on keeping the variant for the reason mentioned in the comment.

Reply:
Done